### ISSUE TITLE
Unit Tests?

### ISSUE THREAD

**Issue by mikecann** [lucasmafra/type-dynamo#2]
Hey,

This is a really nice looking library, im very impressed with your usage of Typescript, but im just wondering about the lack of automated tests. Was there a specific reason why you omitted those?

Thanks again, its an awesome bit of work.

**Comment by lucasmafra** [comment]
Hi @mikecann,

Thanks for asking. To be honest, this is just a pet project. I was working with DynamoDB a couple months ago and I noticed that there is no good ORM for DynamoDB in Typescript. As I was trying to develop my skills with the language I tought it could be fun to develop a typed library for DynamoDB that could help me developing my DynamoDB backed applications safer and faster (btw I hate the AWS Dynamo SDK). As you can see by the readme, at the time I was very excited about it (LoL), but I haven't worked on this in the last 3 months.

**Comment by lucasmafra** [comment]
I'm considering to get back with this lib in the next weeks, but there is really no serious roadmap for now. A couple days ago the guys from the company I was working for reached me to know what's the current status of the library as they are considering to start using in their next project, maybe that will do as an "extra motivation" for me

**Comment by lucasmafra** [comment]
Oh, and I'm sorry I didn't mention any of this in the README. I wasn't expecting people to find this library at all. 😅 I'm going to update the docs to be more explicit about the current status of this.

**Comment by mikecann** [comment]
> I was working with DynamoDB a couple months ago and I noticed that there is no good ORM for DynamoDB in Typescript. 

Yes you are quite right, I am shocked that there is no real good option. There are tons of options for Relational databases but nothing for Dynamo DB. 

From what I can see the only two options are yours (9 stars) and [Dynamo-type-orm](https://github.com/balmbees/dynamo-typeorm) (33 stars) so not exactly a massive selection.

I do prefer the look of your library. As I say I think you have done a fantastic job with the API, it looks great and your readme is really clear. 

I am currently evaluating the options for a project we are working on. Like I mentioned at the top of this issue that a suite of tests would go a long way in making this feel less like a "pet" project and more like a serious offering.
